### ISSUE TITLE
Combine literature and table references

### ISSUE THREAD

**Issue by aflaus** [af-lab/histone-catalogue#45]
At the end of first paragraph of Functional Perspectives section in Introduction there is a literature reference juxtaposed with a table reference. Can these be combined as a single reference within a single parenthesis? No problem if not.

**Comment by carandraug** [comment]
Yes, this can be done. But then, should I use the brackets style for bibliography references or the brackets style for in document tables and figures references?

**Comment by aflaus** [comment]
Bibliography reference I guess.

**Comment by carandraug** [comment]
Ok, then this could be done with:

    \citetext{\citealp{MarzluffNatRevGen2008},
          \tref{tab:typical-histone-differences}}

which then becomes:

> highly conserved 3’ stem-loop [48, Table 1].

on the default build which to me suggest the reference is Table 1 of reference 48.

The command `citetext` allows a citation with any text rather than an actual bibliography key.  The command `citealp` would be the same as using `citep` but removes the paratheses.  However, the separator between multiple citations is a natbib configuration so this would break if it changes.  Also, there is the super style which puts references in superscript (like nature), and in such case, the table reference would not be in superscript.

Not sure how to fix it in any other way though.  Maybe changing the sentence so that the reference to Table 1 is used somewhere else? Actually, maybe the reference to Table 1 is not actually required at this point?
